Sprint-2/debug/author.js

```diff
@@ -11,6 +11,6 @@ const author = {
   alive: true,
 };
 
-for (const value of author) {
-  console.log(value);
+for (const key in author) {
```

Comment:
The error occurs because for...of iterates over the values of iterable objects like arrays, maps, and strings, whereas for...in iterates over the enumerable properties (keys) of an object.